__tests__/app.test.js

```diff
@@ -30,97 +31,83 @@ describe('app routes', () => {
 
     test('returns discs', async() => {
 
-      const expectation = [
-        {
-          id: 1,
-          disc: 'Aviar',
-          speed: 2,
-          type: 'Putt/Approach',
-          brand: 'innova',
-          stable: true,
-          plastics: 'star, gstar, champion, xt, r-pro, glow, dx'
-        },
-        {
-          id: 2,
-          disc: 'Roc',
-          speed: 4,
-          type: 'Midrange',
-          brand: 'innova',
-          stable: true,
-          plastics: 'star, pro, glow, dx'
-        },
-        {
-          id: 3,
-          disc: 'Essence',
-          speed: 8,
-          type: 'Fairway Driver',
-          brand: 'discmania',
-          stable: false,
-          plastics: 'neo, geo'
-        },
-        {
-          id: 4,
-          disc: 'Beast',
-          speed: 10,
-          type: 'Distance Driver',
-          brand: 'innova',
-          stable: true,
-          plastics: 'star, gstar, champion, blizzard, pro, glow, dx'
-        },
-      ];
+      const expectation = discsData.map(disc => disc.disc);
+      const expectedShape = {
```

Comment:
because you had your ids already in your expectation, you can just leave your test as is (as opposed to changing it to match mine).